Reviews icons update (#107)
reef-technologies/handbook: agreements.md

2. Other team members may react to the message in the following ways:

> - ⌛ (`:hourglass:`) – “I am busy and cannot help you right now – but I'll come back as soon as possible”,

Reviewer: ```suggestion
- 👋 (`:wave:`) – “I  cannot help you right now”,
```
I would keep it vague to make it universal
also :hourglass: IMO can be misinterpreted like you are supposed to wait for something, but you really shouldn't be waiting

- Only one developer should perform a code review for each code snippet posted to the `#review` channel.
  Therefore, if someone has already posted the eyes emoji, other staff members should ignore the post and carry on with their work.
- Any team member may review any post – there is no preference as to who should do that.

> - If your PR wasn't reviewed before the daily standup – either someone should volunteer or be picked for the PR.

Reviewer: ```suggestion
- If your PR wasn't marked with 👀 or ✅ before daily standup you must pick Code Reviewer during the daily.
```
